test/redux/alert_test.js

```diff
@@ -59,6 +59,39 @@ describe("alert", () => {
       })
     })
 
+    describe("when the action is SHOW_STAGE_HELP", () => {
+      const initialState = { headerText: "Warning!", bodyText: "You're being watched." }
+      const stageConfigs = {
+        daybreak: {
+          help: {
+            headerText: "I need help!",
+            bodyText: "Tell me what to do",
+          },
+        },
+        stageLackingHelpConfig: {},
+      }
+
+      deepFreeze(initialState)
+      deepFreeze(stageConfigs)
+
+      describe("when the given stage has a help in the given configuration map", () => {
```

Comment:
a thought: "do we need a test to handle the other case?"

Reply:
Well, the other case will return an `undefined`. I can make the reducer return `null` instead and then the `Alert` component should just not render if it hits that condition.

Reply:
i believe the `Alert` component already does a null render when there's no `alert` object passed as a prop, so we're good there